techniques/html/H99.html

```diff
@@ -51,64 +51,64 @@
+<p>ユーザエージェントは、特に電子書籍形式の場合、ページリストを直接レンダリングするのではなく、専用のインタフェースを提供する場合がある。例えば、利用者が移動したいページ番号を入力するための入力ボックスが提示される場合がある。</p></div>
 </section>
 	
 
 
 		
 		<section id="description">
-			<h2>Description</h2>
+			<h2>解説</h2>
 
-			<p>The objective of this technique is to ensure that a page-selection mechanism is provided that makes available programmatically all set page locations in the digital publication.</p>
+			<p>このテクニックの目的は、デジタル出版物内の全ての設定されたページ位置をプログラムによる利用が可能にするページ選択メカニズムを確実に提供することである。</p>
 
-			<p>The page-selection mechanism provides a way to locate static page break locations (i.e., where each new page begins) within a digital publication. These locations typically come from a print equivalent of the edition and allow synchronization in, for example, a mix of print and digital publications in environments such as schools and universities.</p>
+			<p>ページ選択メカニズムは、デジタル出版物内の静的な改ページ位置 (つまり、それぞれの新しいページの開始位置) を特定する方法を提供する。この位置は通常、印刷版と同等な版に由来し、例えば学校や大学などの環境において、印刷版とデジタル版が混在する環境で同期を可能にする。</p>
         
-			<p>The source of pagination is not limited to print but could match up to any statically paginated alternative, such as a <abbr title="Portable Document Format">PDF</abbr> or even a fixed-layout EPUB (i.e., where the content does not reflow to fit the available screen size).</p>
+			<p>ページ番号のソースは印刷物に限定されるものではなく、<abbr title="Portable Document Format">PDF</abbr> や固定レイアウト EPUB (つまり、コンテンツが利用可能な画面サイズに合わせてリフローしないもの) など、静的にページ番号が付けられた代替手段と一致する可能性がある。</p>
         
-			<p>The order of the page break locations in a digital publication is not sequential in all cases, and not all page breaks may be represented. Publishers often rearrange and remove content from the paginated source content when creating a digital version.</p>
+			<p>デジタル出版物内の改ページ位置の順序は、全てのケースで連続しているわけではなく、全ての改ページが表現されるとは限らない。出版社は、デジタル版を作成するときに、ページ番号が付けられたソースコンテンツからコンテンツを再配置したり削除したりすることがよくある。</p>
       
-			<p>Page lists are a central feature of many digital publishing formats. <a href="https://www.w3.org/publishing/epub3/epub-packages.html#sec-nav-pagelist">EPUB 3 defines the page list</a> in its navigation document while EPUB 2 and DAISY 3 use the <a href="http://www.daisy.org/z3986/2005/Z3986-2005.html#NCXElem">pageList element in the NCX</a>. The <a href="https://www.w3.org/TR/dpub-aria-1.0/#doc-pagelist">DPUB-ARIA module</a> also includes a <code class="language-xml">doc-pagelist</code> role for identifying the navigation element containing the page list.</p>
+			<p>ページリストは、多くのデジタル出版フォーマットの中心的な機能である。 EPUB 3ではナビゲーションドキュメント内で<a href="https://www.w3.org/publishing/epub3/epub-packages.html#sec-nav-pagelist">ページリストを定義している</a>が、EPUB 2 及び DAISY 3 では <a href="http://www.daisy.org/z3986/2005/Z3986-2005.html#NCXElem">NCX の pageList 要素</a>を使用している。<a href="https://www.w3.org/TR/dpub-aria-1.0/#doc-pagelist">DPUB-ARIA module</a> はまた、ページリストを含むナビゲーション要素を識別するための <code class="language-xml">doc-pagelist</code> ロールも含まれている。</p>
```

Comment:
最後のセンテンスですが、「DPUB-ARIA module は...」とあるので、「含んでいる」と結んでみました。

```suggestion
			<p>ページリストは、多くのデジタル出版フォーマットの中心的な機能である。 EPUB 3ではナビゲーションドキュメント内で<a href="https://www.w3.org/publishing/epub3/epub-packages.html#sec-nav-pagelist">ページリストを定義している</a>が、EPUB 2 及び DAISY 3 では <a href="http://www.daisy.org/z3986/2005/Z3986-2005.html#NCXElem">NCX の pageList 要素</a>を使用している。<a href="https://www.w3.org/TR/dpub-aria-1.0/#doc-pagelist">DPUB-ARIA module</a> はまた、ページリストを含むナビゲーション要素を識別するための <code class="language-xml">doc-pagelist</code> ロールも含んでいる。</p>
```